website/package.json

```diff
@@ -10,8 +10,8 @@
     "lintDocs": "prettier ../docs/spec/**.md ../docs/related/**.md --write"
   },
   "dependencies": {
-    "@docusaurus/core": "2.0.0-alpha.61",
-    "@docusaurus/preset-classic": "2.0.0-alpha.61",
+    "@docusaurus/core": "^2.0.0-alpha.61",
```

Comment:
They do not guarantee breaking changes on those versions so using pinned version might be better

Reply:
Ok, pinned to next alpha